Inventory_Management_System/InventoryManagement/Inventory.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Inventory_Management_System.ProductManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Inventory_Management_System.InventoryManagement
+{
```

Comment:
Could you try using new namespace syntax out in c#10
 (namespace Inventory_Management_System.InventoryManagement;) no need for the brackets